src/CattyTests/DataModel/VariablesContainerTest.swift

```diff
@@ -96,4 +96,172 @@ final class VariablesContainerTest: XCTestCase {
         XCTAssertEqual(1, container.allLists(for: objectA)?.count)
         XCTAssertEqual(1, container.allLists(for: objectB)?.count)
     }
+
+    func testSpriteObjectForObjectVariableWhenNotList() {
+        let objectA = SpriteObject()
+        objectA.name = "testObjectA"
+
+        let objectB = SpriteObject()
+        objectB.name = "testObjectB"
+
+        let userVariable1 = UserVariable()
```

Comment:
Please use the newly added initializer for the `UserVariable` (with `name` and `isList`).